lib/init.js

```diff
@@ -79,6 +79,6 @@ module.exports = function (opts, callback) {
       );
     }
 
-    callback(err);
+    callback(err, results);
```

Comment:
It's not used yet but the function signature in the doc string does declare it. 
https://github.com/formidablelabs/denim/blob/master/lib/init.js#L21

Also, if you're building tooling on top of denim (as we are with the dashboard) it's useful to expose the results for any additional processing.